src/modules/site-v2/base/views/tools/genetic_mapping.py

```diff
@@ -209,10 +197,9 @@ def report(id):
 
     'id': id,
 
-    # URLs
-    'report_url': report_url,
-    'report_status_url': url_for("genetic_mapping.report_status", id=id),
-    'data_download_url': data_download_url,
+    # Links to the input data file and report output files, if they exist

```

Comment:
The new `Report` class tracks where input/output are stored, so we can use it to get the relevant filepaths (and to check if files exist at those paths)